### PR TITLE
uwsgi: Add user/group for uwsgi service.

### PR DESCRIPTION
**Pull request by oconnorr**
Also add a uwsgi directory under /run (defaulting to /run/uwsgi) where the uwsgi user can place sockets.

### CODE REVIEW

File: nixos/modules/services/web-servers/uwsgi.nix [hunk 1]
```diff
@@ -89,24 +95,47 @@ in {
...
+        type = types.str;
+        default = "uwsgi";
+        description = "Group account under which nginx runs.";
+      };
+    };
   };
 
-  config = mkIf cfg.enable {
-
+  config = mkIf cfg.enable (let
```
Comment (oconnorr) [inline nixos/modules/services/web-servers/uwsgi.nix]: Remove useless let

File: nixos/modules/services/web-servers/uwsgi.nix [hunk 2]
```diff
@@ -89,24 +95,46 @@ in {
         description = "Plugins used with uWSGI";
       };
 
-    };
+      user = mkOption {
+        type = types.str;
+        default = "uwsgi";
+        description = "User account under which nginx runs.";
```
Comment (lethalman) [inline nixos/modules/services/web-servers/uwsgi.nix]: Not necessarily nginx, say web server or whatelse.

Comment (oconnorr) [inline nixos/modules/services/web-servers/uwsgi.nix]: Oops.  Thanks for catching this.

**Comment by lethalman** [comment]
Looks good to me.

**Pull request merged**